exercises/circular-buffer/circular_buffer_test.py

```diff
@@ -7,40 +7,37 @@
 )
 
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.1
```

Comment:
Minor formatting point, but can you add a blank line after the version string for consistency?

Reply:
Done